usaspending_api/api_contracts/contracts/v2/reporting/agencies/submission_history/agency_code/fiscal_year/fiscal_period.md

```diff
@@ -1,7 +1,7 @@
 FORMAT: 1A
 HOST: https://api.usaspending.gov
 
-# Agencies Reporting Publish Dates History [/api/v2/reporting/submission_history/{agency_code}/{fiscal_year}/{fiscal_period}]
+# Agencies Reporting Publish Dates History [/api/v2/reporting/agencies/submission_history/{agency_code}/{fiscal_year}/{fiscal_period}]
```

Comment:
The original endpoint structure of separating this endpoint away from the agencies dir was intentional since it is a little different than the others. But I can see the justification of grouping it under agencies. It could also be better defined as `/api/v2/reporting/submissions/submission_history/{agency_code}/{fiscal_year}/{fiscal_period}`

Open to discussion on what is the most intuitive

Reply:
I thought it made more sense putting it under `agencies` since we are requiring an agency code, so it would have to be agency specific. I am fine with moving it to a separate `submissions` category if there is a meaningful difference in how they are being used.

Reply:
If that is the case, wouldn't it be more consistent to rearrange the endpoint to

`/api/v2/reporting/agencies/{agency_code}/{fiscal_year}/{fiscal_period}/submission_history/`

Reply:
That works for me